docs/installation/application-configuration.md

```diff
@@ -16,33 +16,64 @@ For operating Shopsys Platform it is needed to have correctly set connections to
-| `DATABASE_PASSWORD`                   | `'root'`                           | ...                                                                                                                             |
-| `ELASTICSEARCH_HOST`                  | `'elasticsearch:9200'`             | host of your Elasticsearch, you can use multiple hosts like `'["elasticsearch:9200", "elasticsearch2:9200"]'`                   |
-| `REDIS_HOST`                          | `'redis'`                          | host of your Redis storage (credentials are not supported right now)                                                            |
-| `REDIS_PREFIX`                        | `''`                               | separates more projects that use the same redis service                                                                         |
-| `MAILER_DSN`                          | `smtp://smtp-server:25`            | set to `null://null` if you don't want to send any emails, see https://symfony.com/doc/current/mailer.html#disabling-delivery   |
-| `APP_SECRET`                          | `'ThisTokenIsNotSoSecretChangeIt'` | randomly generated secret token                                                                                                 |
-| `ELASTIC_SEARCH_INDEX_PREFIX`         | `''`                               | separates more projects that use the same elasticsearch service                                                                 |
-| `IGNORE_DEFAULT_ADMIN_PASSWORD_CHECK` | `'0'`                              | set to `true` if you want to allow administrators to log in with default credentials                                            |
-| `OVERWRITE_DOMAIN_URL`                | `'http://webserver:8080'`          | overwrites URL of all domains for acceptance testing (set to `~` to disable)                                                    |
-| `SELENIUM_SERVER_HOST`                | `'selenium-server'`                | with native installation the selenium server is on `localhost`                                                                  |
-| `SHOPSYS_CONTENT_DIR_NAME`            | `'content-test'`                   | web/content-test/ directory is used instead of web/content/ during the tests                                                    |
-| `TRUSTED_PROXIES`                     | `'127.0.0.1'`                      | proxies that are trusted to pass traffic, used mainly for production (set as text separated by comma for multiple values)       |
-| `CDN_DOMAIN`                          | `'//'`                             | specifies URL of a Content Delivery Network (CDN) that is used to serve static assets such as images, CSS, and JavaScript files |
+`DATABASE_HOST` (default: `'postgres'`)
+: access data of your PostgreSQL database
+
+`DATABASE_PORT` (default: `null`)
+: ...
+
+`DATABASE_NAME` (default: `'shopsys'`)
+: ...
```

Comment:
What was the motivation for this change? I find previous version to be more clear